app/assets/stylesheets/success_stories.css.scss

```diff
@@ -0,0 +1,25 @@
+.success-stories-photo {
+  border-radius: 50%;
+  height: 120px;
+  width: 120px;
+}
+
+.success-stories-photo--left {
+  margin-left: 15px;
+}
+
+.success-stories-photo--right {
```

Comment:
I was using the BEM (Block-Element-Modifier) CSS naming-syntax... where `--` is sometimes used for "modifier" classes.  I've also seen this used in a system known as "OOCSS" (Object-Oriented CSS) There are lots of variations & ideas on these systems (and on CSS naming in general) - but this is one system I've typically found easy to use and maintain: use the initial page/partial/template as the initial namespace / 'Block' (`success-stories`), then use a semantic term describing the unique child item you're assigning a CSS rule to as the Element - (a `photo` on the page).  Last `--right` or `--left` would be a modified instance of the block-element `success-stories-photo` - one that is displayed to the right or left.

I prefer using the `--` to explicitly precede modifiers, that way they aren't confused with the nested namespacing (which is single hyphen-delimited).   IMO each 'namespace' part of the BEM syntax is also easier to read w/o containing internal hyphens (like how `success-stories` contains an internal hyphen) - but it's sometimes hard to avoid - so this is a bad example of that.